scripts/build-docker.sh

```diff
@@ -108,7 +108,14 @@ EOF
 }
 
 if [[ "$(has_local_image)" -gt 0 ]]; then
-  docker build -t user-service:latest .
+  if [ $(arch) = "arm64" ]
+  then
+      echo "Building for ARM"
+      docker build --platform=linux/amd64 -t user-service:latest .
```

Comment:
should the platform be `linux/arm64` instead of `linux/amd64` here?

Reply:
Unfortunately, most of these images doesn't build natively on ARM, so what we do here is tell docker to build them for `linux/amd64`